src/greeklt/greek_elot_transliteration.py

```diff
@@ -0,0 +1,231 @@
+def greek_elot_transliteration(string):
+    from remove_accentuation import remove_accentuation
+#    reference_string = string
+    string = remove_accentuation(string, 1)
+    lowercase = {
+        'α': 'a',
+        'β': 'v',
+        'γ': 'g',
+        'δ': 'd',
+        'ε': 'e',
+        'ζ': 'z',
+        'η': 'i',
+        'θ': 'th',
+        'ι': 'i',
+        'κ': 'k',
+        'λ': 'l',
+        'μ': 'm',
+        'ν': 'n',
+        'ξ': 'x',
+        'ο': 'o',
+        'π': 'p',
+        'ρ': 'r',
+        'σ': 's',
+        'τ': 't',
+        'υ': 'y',
+        'φ': 'f',
+        'χ': 'ch',
+        'ψ': 'ps',
+        'ω': 'o'
+    }
+    caps = {
+        'Α': 'A',
+        'Β': 'V',
+        'Γ': 'G',
+        'Δ': 'D',
+        'Ε': 'E',
+        'Ζ': 'Z',
+        'Η': 'I',
+        'Θ': 'TH',
+        'Ι': 'I',
+        'Κ': 'K',
+        'Λ': 'L',
+        'Μ': 'M',
+        'Ν': 'N',
+        'Ξ': 'X',
+        'Ο': 'O',
+        'Π': 'P',
+        'Ρ': 'R',
+        'Σ': 'S',
+        'Τ': 'T',
+        'Υ': 'Y',
+        'Φ': 'F',
+        'Χ': 'CH',
+        'Ψ': 'PS',
+        'Ω': 'O'
+    }
+    # Simple digraphs with no extra rules or edge cases
+    # No need for mixed casing "Γγ" or "Γξ" against Greek phonology
+    el_simple_digraphs = [
+        'γγ',
+        'γξ',
+        'γχ',
+        'ου'
+    ]
+    eng_simple_digraphs = [
+        'ng',
+        'nx',
+        'nch',
+        'ou'
+    ]
+    el_simple_cap_digraphs = [
+        "ΓΓ",
+        "ΓΞ",
+        "ΓΧ",
+        "ΟΥ"
+    ]
+    eng_simple_cap_digraphs = [
+        "NG",
+        "NX",
+        "NCH",
+        "OU"
+    ]
+
+    el_mono_digraph_sub = [
+        "TH",
+        "CH",
+        "PS"
+    ]
+#   Accent based digraphs
+#    el_low_acc_digraphs = [
+#        "άυ",
+#        "αϋ",
+#
+#        "έυ",
+#        "εϋ",
+#
+#        "ήυ",
+#        "ηϋ"
+#
+#    ]
+#    el_mix_acc_digraphs = [
+#        "Άυ",
+#        "Αϋ",
+#
+#        "Έυ",
+#        "Εϋ",
+#
+#        "Ήυ",
+#        "Ηϋ"
+#    ]
+#    el_cap_acc_digraphs = [
+#        "ΆΥ",
+#        "ΑΫ",
+#
+#        "ΈΥ",
+#        "ΕΫ",
+#
+#        "ΉΥ",
+#        "ΗΫ"
+#    ]
```

Comment:
I suggest we delete all commented code since it's not used!

Reply:
Hmm... Reminder to myself not to use GitHub on the phone again. It had me fooled, thinking I only had one notification haha

Sure! I'll keep it saved in a file or something on my own computer for future use.